shoplist/src/main/java/com/netguru/codereview/ui/MainFragment.kt

```diff
@@ -0,0 +1,65 @@
+package com.netguru.codereview.ui
+
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.ProgressBar
+import android.widget.Toast
+import androidx.core.view.isVisible
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import coil.load
+import com.netguru.codereview.network.model.ShopListItemResponse
+import com.netguru.codereview.network.model.ShopListResponse
+import com.netguru.codereview.shoplist.R
+import com.netguru.codereview.ui.model.ShopList
+import javax.inject.Inject
+
+class MainFragment : Fragment() {
+
+    @Inject
```

Comment:
As there is no dependency injection used, no need to use Inject annotation
```suggestion
    //@Inject
```